cypress_test/integration_tests/desktop/calc/top_toolbar_spec.js

```diff
@@ -17,6 +17,60 @@ describe('Top toolbar tests.', function() {
 		helper.afterAll(testFileName, this.currentTest.state);
 	});
 
+	function getTextEndPosForFirstCell() {
+		calcHelper.dblClickOnFirstCell();
+
+		helper.getCursorPos('left', 'currentTextEndPos');
+	}
+
+	it('Merge cells', function() {
+
+		// Select the full column
+		calcHelper.selectFirstColumn();
+	
+		// Despite the selection is there, merge cells needs more time here.
+		cy.wait(1000);
+	
+		cy.get('.w2ui-tb-image.w2ui-icon.togglemergecells')
+			.click();
+
+		desktopHelper.checkDialogAndClose('Merge Cells');
+	});
+
+	it('Enable text wrapping.', function() {
+		calcHelper.clickOnFirstCell(true, false);
+		calcHelper.typeIntoFormulabar('_This_is_a_really_long_text');
+		helper.initAliasToNegative('originalTextEndPos');
+
+		getTextEndPosForFirstCell();
+		cy.get('@currentTextEndPos')
+			.as('originalTextEndPos');
+
+		cy.get('@currentTextEndPos')
+			.should('be.greaterThan', 0);
+
+		calcHelper.selectFirstColumn();
+
+		cy.get('.w2ui-tb-image.w2ui-icon.wraptext')
+			.click();
```

Comment:
If I comment out these lines above, where the test applies the text wrapping, the test still passes. That means we don't actually verify that the text wrapping toolbar item does work. It would be good to find out a way to test it properly.